### ISSUE TITLE
Content of notification not showing in the notification timeline

### ISSUE THREAD

**Issue by maddler** [trunks-social/trunks-public#7]
Aftger receiving a notification, the notification timeline will not always show the content (e.g. message, new follower, etc.) until manually refreshed.

**Comment by neatchee** [comment]
@maddler
We're going through and cleaning up the issue tracker!

Can you please confirm that:
a) This bug is still occurring as of 2023-07-04
b) Whether you've been able to identify the circumstances in which the notification timeline is updated vs when it is not

Thanks!

**Comment by maddler** [comment]
Yes, it seems to be still happening. I get the notification but need to refresh once in the notification section in order to see the actual message. It seems to be happening for most of the notifications. 

Thanks. 

________________________________
